students/Mintae_K/lesson06/test_mailroom.py

```diff
@@ -0,0 +1,22 @@
+from mailroom4 import *
+import os.path
+
+donors = {'William Gates, III': [54842.49, 48965.25],
+            'Mark Zuckerberg': [7852.25, 48652.0, 3548.0],
+            'Jeff Bezos': [5486.0, 58794.02, 7412.1],
+            'Paul Allen': [46872.02]}
+
+
+def test_check_name_exist():
+    (yes_exist, s) = check_name_exist('Starbucks Coffee', donors)
+    assert 0 is yes_exist
```

Comment:
why aren't you asserting on `s`? and best to use good variable name so that the reader can understand the code, `found_message` is better than `s`